config/sync/user.role.contributor.yml

```diff
@@ -19,6 +19,7 @@ permissions:
   - 'access video_browser entity browser pages'
   - 'add citation entities'
   - 'administer main menu items'
+  - 'break content lock'
```

Comment:
are we sure we want to let contributors break the locks?  Maybe we ought to reserve that for higher level roles only?  I'm not married to the idea, but it's something worth considering.

Reply:
There's not really much concern in my mind. It's not really dangerous it's just annoying to users when they simultaneously edit which is rare in the first place.